### ISSUE TITLE
(ADMIN) 'make everyone antag' also antags antagbanned players

### ISSUE THREAD

**Issue by Landerlow** [ParadiseSS13/Paradise#12906]
**Issue Description**:
The option for admins to make everyone antags makes antagbanned players antag. 

**What did you expect to happen**: 
Not to have antagbanned players get antag status

**What happened instead**:
Antagbanned player(s) got antag status

**Why is this bad/What are the consequences**:
This isn't supposed to happen and could get antagbanned people falsely into trouble. 

**Steps to reproduce the problem**:
Be admin
Make everyone antag secret

**When did the problem start happening**:
Just now. 1/14/20 - may always ahve been happening

**Extra information**:

**Comment by Birdtalon** [comment]
Relevant code if you want to add a check, although what would you suggest to do with the antag banned players?

https://github.com/ParadiseSS13/Paradise/blob/076ef673cda2630eed893db11eb27154a78d56ee/code/modules/admin/topic.dm#L2694-L2715

**Comment by Landerlow** [comment]
I suggest they would be not made antags when using the button. I know it says 'everyone' but .. still.

**Comment by taukausanake** [comment]
I agree with @Landerlow, you can't let them become antags. It's possible for them to meta that everyone is an antag when they, banned from being antag, becomes one. You have to restrict them always.

You _could_ also make them the top priority for kill targets by the rest or part of the new antagged crew. That could be fun

**Comment by Spacemanspark** [comment]
I don't think we need to turn banned players into people wearing public 'dunce' hats. Prioritizing them for death is fairly shitty.

I don't think they should be becoming antagonists, however.
